# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/base/crashtests/crashtests.list
# Commit: 21460b29ead8
# Full Hash: 21460b29ead88c28da41e02f331779324a3eb559
# Author: Botond Ballo <botond@mozilla.com>
# Date: 2022-03-15 05:30:53
# Regressor Bug: 1749190
# File Overlap Count: 2
# Description:
#   Bug 1749190 - Add a crashtest. r=tnikkel
#   
#   Depends on D137944
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D137945
# ==============================================================================

diff -r 3d623418ee05 -r 21460b29ead8 layout/base/crashtests/crashtests.list
--- a/layout/base/crashtests/crashtests.list	Tue Mar 15 00:56:00 2022 +0000
+++ b/layout/base/crashtests/crashtests.list	Tue Mar 15 00:56:00 2022 +0000
@@ -563,6 +563,7 @@
 load 1729578.html
 load 1729581.html
 load 1734007.html
+asserts-if(Android,0-1000) load 1749190.html
 load 1745860.html
 pref(layout.accessiblecaret.enabled,true) load 1746989.html
 load 1752649.html
